### PR TITLE
Replace metacity with gnome-kiosk

### PR DESCRIPTION
**Pull request by rvykydal**
This PR replaces metacity with gnome-kiosk in installer image

### CODE REVIEW

File: data/window-manager/config/Makefile.am [hunk 1]
```diff
@@ -30,7 +29,6 @@ install-data-hook:
 	$(MKDIR_P) $(DESTDIR)$(schemadir)
 	$(LN_S) -f /usr/share/glib-2.0/schemas/org.gnome.desktop.wm.keybindings.gschema.xml $(DESTDIR)$(schemadir)
 	$(LN_S) -f /usr/share/glib-2.0/schemas/org.gnome.desktop.wm.preferences.gschema.xml $(DESTDIR)$(schemadir)
-	$(LN_S) -f /usr/share/glib-2.0/schemas/org.gnome.metacity.gschema.xml $(DESTDIR)$(schemadir)
 	$(LN_S) -f /usr/share/glib-2.0/schemas/org.gnome.metacity.enums.xml $(DESTDIR)$(schemadir)
```
Comment (poncovka) [inline data/window-manager/config/Makefile.am:32]: I guess that we should remove also this line.

**Review by poncovka — APPROVED** [review]
Otherwise, it looks good to me. Thanks!

File: data/window-manager/config/Makefile.am (hunk 1 above)
Comment (rvykydal) [reply data/window-manager/config/Makefile.am]: Added

**Review by jkonecny12 — APPROVED** [review]
Looks great to me. Thanks a lot!

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Review by poncovka — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**